Configuration/Settings.yaml

```diff
@@ -34,6 +34,10 @@ TYPO3:
         reference:
           indexing: "${value.identifier}"
 
+        'TYPO3\Media\Domain\Model\AssetList':
```

Comment:
Must be array<TYPO3\Media\Domain\Model\Asset> and additional config for single asset is missing I think.